internal/cli/atlas/metrics/databases/describe.go

```diff
@@ -59,14 +59,22 @@ func (opts *DatabasesDescribeOpts) Run() error {
 	return opts.Print(r)
 }
 
-// mcli om metric(s) disk(s) describe <hostId:port> <name> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
-func DatabasesDescribeBuilder() *cobra.Command {
+// mcli atlas metric(s) database(s) describe <host:port> <databaseName> --granularity g --period p --start start --end end [--type type] [--projectId projectId].
+func DescribeBuilder() *cobra.Command {
 	const argsN = 2
-	opts := &DatabasesDescribeOpts{}
+	opts := &DescribeOpts{}
 	cmd := &cobra.Command{
-		Use:   "describe <hostId> <name>",
+		Use:   "describe <host:port> <databaseName>",
 		Short: "Describe database measurements for a given host database.",
 		Args:  require.ExactArgs(argsN),
+		Annotations: map[string]string{
+			"args":              "hostname:port,databaseName",
+			"requiredArgs":      "hostname:port,databaseName",
+			"hostname:portDesc": "Hostname and port number of the machine running the Atlas MongoDB process.",
+			"databaseNameDesc":  "Label that identifies the database from which you want to retrieve measurements.",
+		},
+		Example: `
+  $ mongocli atlas metrics database describe atlas-lnmtkm-shard-00-00.ajlj3.mongodb.net:27017 Cluster0 --granularity PT1M --period P1DT12H`,
```

Comment:
It would be a good idea to add an explanation
```suggestion
		Example: `  Retrieve database metrics for the process running on atlas-lnmtkm-shard-00-00.ajlj3.mongodb.net:27017
  $ mongocli atlas metrics database describe atlas-lnmtkm-shard-00-00.ajlj3.mongodb.net:27017 Cluster0 --granularity PT1M --period P1DT12H`,
```

Reply:
One addition: Start the sentence on line 76 with:
` This example retrieves database metrics for ....

Also, do we explain elsewhere what is granularity and period, and how to specify these values? Assume yes, but I don't recall seeing them.